database_admin/migrations/056_mark_stale_fix.down.sql

```diff
@@ -0,0 +1,85 @@
+CREATE OR REPLACE FUNCTION delete_systems(ids INT[])
+    RETURNS INTEGER
+AS
+$$
+DECLARE
+    system_ids INT[];
+    tmp_cnt    INTEGER;
+BEGIN
+    system_ids := ARRAY(
+            SELECT id
+            FROM system_platform
+            WHERE id = ANY (ids)
+            ORDER BY rh_account_id, id FOR UPDATE OF system_platform
+        );
+
+    UPDATE system_platform sp
+    SET opt_out = true
+    WHERE id = ANY (ids);
+
+    GET DIAGNOSTICS tmp_cnt = row_count;
+    RAISE NOTICE 'Marked systems %', text(tmp_cnt);
+
+
+    DELETE
+    FROM system_advisories
+    WHERE system_id = ANY (system_ids);
+
+    GET DIAGNOSTICS tmp_cnt = row_count;
+    RAISE NOTICE 'Deleted system_advisories %', text(tmp_cnt);
+
+
+    DELETE
+    FROM system_repo
+    WHERE system_id = ANY (system_ids);
+
+    GET DIAGNOSTICS tmp_cnt = row_count;
+    RAISE NOTICE 'Deleted system_repos %', text(tmp_cnt);
+
+
+    DELETE
+    FROM system_package
+    WHERE system_id = ANY (system_ids);
+
+    GET DIAGNOSTICS tmp_cnt = row_count;
+    RAISE NOTICE 'Deleted system_packages %', text(tmp_cnt);
+
+
+    DELETE
+    FROM system_platform
+    WHERE id = ANY (system_ids);
+
+    GET DIAGNOSTICS tmp_cnt = row_count;
+    RAISE NOTICE 'Deleted system_platform %', text(tmp_cnt);
+
+
+    RETURN tmp_cnt;
+END
+$$ LANGUAGE plpgsql;
+
+DROP FUNCTION mark_stale_systems();
+
+CREATE OR REPLACE FUNCTION mark_stale_systems(max_systems integer)
```

Comment:
The "down" part of migration should include previous version of mark_stale_systems - without "max_systems" arg. And the version with an arg should be dropped if it exists.